### PR TITLE
[jspdf] Fix setting colors using an HTML color string.

### PR DESCRIPTION
**Pull request by pcorpet**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/MrRio/jsPDF/blob/6c6ca0c4554eb5b6446dab9337befd6625119bee/types/index.d.ts#L552>
- [X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/jspdf/jspdf-tests.ts [hunk 1]
```diff
@@ -174,9 +176,9 @@ function test_rect_squares() {
     doc.setDrawColor(0);
     doc.setFillColor(255, 0, 0);
     doc.rect(120, 20, 10, 10, 'FD'); // filled red square with black borders
-    doc.setDrawColor(0);
```
Comment (FrankBrullo) [inline types/jspdf/jspdf-tests.ts]: Why this case was removed? I think this is a valid case with only one number. Please explain.

File: types/jspdf/jspdf-tests.ts [hunk 2]
```diff
@@ -174,9 +176,9 @@ function test_rect_squares() {
...
-    doc.setFillColor(255, 255, 255);
```
Comment (FrankBrullo) [inline types/jspdf/jspdf-tests.ts]: Why this case was removed? I think this is a valid case with three numbers. Please explain.

**Review by FrankBrullo — CHANGES_REQUESTED** [review]

File: types/jspdf/jspdf-tests.ts (hunk 1 above)
Comment (pcorpet) [reply types/jspdf/jspdf-tests.ts]: It's still there few lines above.

File: types/jspdf/jspdf-tests.ts (hunk 2 above)
Comment (pcorpet) [reply types/jspdf/jspdf-tests.ts]: It's still there 3 lines above.

**Review by jemerald — APPROVED** [review]

**Review by FrankBrullo — APPROVED** [review]

**Comment by FrankBrullo** [comment]
@jemerald Once someone starts a review please let him/her approve the pull request without overriding them. Thanks.

**Comment by jemerald** [comment]
> @jemerald Once someone starts a review please let him/her approve the pull request without overriding them. Thanks.

Sorry @FrankBrullo

**Pull request merged**